samtranslator/model/sam_resources.py

```diff
@@ -60,6 +60,16 @@
 MAX_CANARY_UNIQUE_ID_LENGTH = 5
 CANARY_NAME_PREFIX = "sam-"
 
+# The default values for ComparisonOperator, Threshold and Period based on the MetricName provided by the user
+# These default values were acquired from the Create Canary page in the Synthetics Canary dashboard
+DEFAULT_METRIC_VALUES = {
+    "SuccessPercent": {"ComparisonOperator": "LessThanThreshold", "Threshold": 90, "Period": 300},
+    "Failed": {"ComparisonOperator": "GreaterThanOrEqualToThreshold", "Threshold": 1, "Period": 300},
+    "Duration": {"ComparisonOperator": "GreaterThanThreshold", "Threshold": 30000, "Period": 900},
+}
```

Comment:
We might need a maintenance runbook for this if there is no automated solution. I think it is something we want to re-visit later @c2tarun

Reply:
Yes definitely, we'll add a runbook. Until then @aaythapa can you please add link to metrics published by Synthetics in comments here: https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Synthetics_Canaries_metrics.html